### PR TITLE
Changed some $replace rules to $redirect #83637

### PR DESCRIPTION
**Pull request by AdamWr**
https://github.com/AdguardTeam/AdguardFilters/issues/83637

### CODE REVIEW

**Review by Alex-302 — COMMENTED** [review]
But without content type (may be different when using app or browser extension) they cannot be converted to ubo syntax.

**Comment by AdamWr** [comment]
>But without content type (may be different when using app or browser extension) they cannot be converted to ubo syntax.

Okay, added content type.

File: EnglishFilter/sections/foreign.txt [hunk 1]
```diff
@@ -9067,14 +9067,12 @@ gamme.com.tw##a[href^="https://www.gamme.com.tw/redirect.php?id="][target="_blan
 ! https://github.com/AdguardTeam/AdguardFilters/issues/15480
 @@||service.4gtv.tv/4gtv/Data/GetAD.ashx
 @@||imasdk.googleapis.com/js/sdkloader/ima3.js$domain=4gtv.tv
-||vast.doublemax.net/adserver/vast.xml$replace=/(<VAST[\s\S]*?>)[\s\S]*<\/VAST>/\$1<\/VAST>/,important
-||ads.adaptv.advertising.com/*roll&pageUrl=4gtv.tv$replace=/(<VAST[\s\S]*?>)[\s\S]*<\/VAST>/\$1<\/VAST>/,important
-||facebook.com/*/instream/vast.xml?*&pageurl=http*.4gtv.tv$replace=/(<VAST[\s\S]*?>)[\s\S]*<\/VAST>/\$1<\/VAST>/,important
-||pubads.g.doubleclick.net/gampad/ads?*&output=xml_vast*&url=http*.4gtv.tv$replace=/(<VAST[\s\S]*?>)[\s\S]*<\/VAST>/\$1<\/VAST>/,important
+||vast.doublemax.net/adserver/vast.xml$script,xmlhttprequest,other,redirect=noopvast-2.0
```
Comment (Alex-302) [inline EnglishFilter/sections/foreign.txt:9070]: are `.XML` detected as script?

Comment (AdamWr) [reply EnglishFilter/sections/foreign.txt:9070]: I'm not sure. but as I recall, I saw it somewhere with apps, so I prefered to add it.
I think that adding `script` content type to these rules shouldn't cause any problems, as before there were `$replace` rules without any content type.

**Review by zloyden — APPROVED** [review]

File: EnglishFilter/sections/foreign.txt (hunk 1 above)
Comment (Alex-302) [reply EnglishFilter/sections/foreign.txt:9070]: Yep.

**Review by Alex-302 — APPROVED** [review]

**Review by ameshkov — APPROVED** [review]

**Pull request merged**